govc/USAGE.md

```diff
@@ -4185,6 +4195,27 @@ Options:
   -t=false               Include sample times
 ```
 
+## namespace.change
```

Comment:
Is `change` the usual naming and not `update`??

Reply:
There are 9 instances of `update` versus ~20 of `change`
I just decided to stick to the majority

Reply:
iirc, we started with `change` instead of the vmodl1 `reconfigure` naming. Later some commands were named as `update` (more common in vmodl2).  We could make this consistent and rename existing commands outside of this PR. We also have an alias mechanism so the rename would not break to existing scripts.

Reply:
Got it. Switched to `update`